examples_tests/17.SimpleBulletIntegration/Camera.hpp

```diff
@@ -0,0 +1,377 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _CAMERA_IMPL_
+#define _CAMERA_IMPL_
+
+#include <nabla.h>
+#include <iostream>
+#include <cstdio>
+#include <fstream>
+#include <chrono>
+
+using namespace nbl;
+using namespace core;
+using namespace ui;
+
+class Camera { 
+public:
+	Camera( const core::vector3df& position,
+			const core::vectorSIMDf& lookat,
+			float aspect,
+			float fovy = core::radians(60),
+			float znear = 1.0f,
+			float zfar = 3000.0f,
+			bool leftHanded = true
```

Comment:
somehow I'm under the impression that a camera should just take a projection matrix (and there should be a 4x4 projection matrix creation function that takes these)

Reply:
We could also murder `recomputeProjectionMatrix`